openlibrary/templates/type/edition/view.html

```diff
@@ -389,6 +385,9 @@ <h3 class="collapse">$_("ID Numbers")</h3>
 
     </div>
 
+    $ work = page.works and page.works[0]
```

Comment:
Can we make this a little stricter? It will set work to an empty array for orphan works, which might be fine, but having an if statement might be a little cleaner / more secure.

Reply:
Good feedback. will revisit -- this is pretty consistently how our code checks for / fetches the work from the edition currently. If we're going to change this, let's have a branch dedicated towards this change and we can make the change wherever necessary.